src/commands/boardgamegeek.py

```diff
@@ -0,0 +1,105 @@
+import requests
+import xmltodict
+
+from disnake import ApplicationCommandInteraction, Embed, Color
+from disnake.ext.commands import Cog, slash_command
+
+from src.data_files.BoardGameModel import BoardGameModel
+from src.utils.paginator import BoardGamePaginator
+
+
+class BoardGamesCog(Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @slash_command(name="fetchBg", description="Fetch boardgame's information from BGG")
```

Comment:
I think they won't let you have command names with uppercase letters, did this work for you?
Additionally, what do you think about calling it 'bggsearch' or something referencing specifically bgg? In case we add other ways of searching games (igdb for videogames, for example) and people want to know where they are searching.

Reply:
didn't check it tbh. It was a last minute change to make it look better. Renaming to bggsearch